source/distributed/tria_base.cc

```diff
@@ -90,7 +90,7 @@ namespace parallel
   {
     std::size_t mem =
       this->dealii::Triangulation<dim, spacedim>::memory_consumption() +
-      MemoryConsumption::memory_consumption(mpi_communicator) +
+      MemoryConsumption::memory_consumption(this->mpi_communicator) +
```

Comment:
This should go into the base class (except I don't think the variable should actually live in the base class).